# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdepim/hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- kdepim-3.4.2/korn/korncfg.ui.orig	2005-09-20 23:32:00.567000763 +0200
+++ kdepim-3.4.2/korn/korncfg.ui	2005-09-20 23:32:29.729068317 +0200
@@ -162,7 +162,4 @@
     <slot access="protected">slotEditBox()</slot>
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
-<includehints>
-    <includehint>keditlistboxmanager.h</includehint>
-</includehints>
 </UI>
--- kdepim-3.4.2/korn/kornboxcfg.ui.orig	2005-09-20 23:31:00.503159581 +0200
+++ kdepim-3.4.2/korn/kornboxcfg.ui	2005-09-20 23:31:26.266802133 +0200
@@ -1692,7 +1692,6 @@
     <includehint>kurlrequester.h</includehint>
     <includehint>klineedit.h</includehint>
     <includehint>kpushbutton.h</includehint>
-    <includehint>keditlistboxmanager.h</includehint>
     <includehint>keditlistbox.h</includehint>
     <includehint>klineedit.h</includehint>
 </includehints>
--- kdepim-3.4.2/kmail/smimeconfiguration.ui~	2005-05-23 14:11:54.000000000 +0200
+++ kdepim-3.4.2/kmail/smimeconfiguration.ui	2005-09-21 14:59:12.757089481 +0200
@@ -411,6 +411,6 @@
 </connections>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>kleo::keyrequester.h</includehint>
+    <includehint>../certmanager/lib/ui/keyrequester.h</includehint>
 </includehints>
 </UI>
